CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [0.0.3] - 2022-11-02
+
+### Added
+
+- VSphere cluster provider
+- Builder macro for templated values in an agents config
+- EKS cluster provider accepts `eksctl` configuration files
+- ECS cluster provider support for existing IAM instance profiles
+- TestManager support for custom status columns
+- Support for bash agents using `test-agent-cli`
+
+### Changed
```

Comment:
It seems like we're missing some changes here. Some on top of my head:
https://github.com/bottlerocket-os/bottlerocket-test-system/pull/607
https://github.com/bottlerocket-os/bottlerocket-test-system/pull/585

All of the changes since 0.0.2 are listed here. We can skip the dependabot bump PRs.
https://github.com/bottlerocket-os/bottlerocket-test-system/compare/v0.0.2...develop